broker-plugin/broker-cli/src/main/java/io/enmasse/broker/cli/Cli.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2016-2018, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.enmasse.broker.cli;
+
+
+import io.enmasse.amqp.Artemis;
+import io.enmasse.amqp.SyncRequestClient;
+import io.vertx.core.Vertx;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class Cli {
+
+    private static final Logger log = LoggerFactory.getLogger(Cli.class.getName());
+
+    private SyncRequestClient brokerClient;
+    private Artemis artemis;
+    private String certDir = "/opt/apache-artemis/custom/enmasse-certs";
+
+    private void start() {
+
+        Vertx vertx = Vertx.vertx();
+        ClientFactory clientFactory = new ClientFactory(vertx, certDir);
+
+        try {
+            brokerClient = clientFactory.connectBrokerManagementClient("localhost", 5673);
+            artemis = new Artemis(brokerClient);
+
+        } catch (NullPointerException e) {
```

Comment:
In general catching NPE is horrid!  If another part of the code called within this try/catch encounters a NPE, you've mislead the person investigating into looking for a create/connect failure.

Better to handle the connection failed separately.  I think you just want a if (brokerClient == null) { throw nice meaningful exception}